libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1100,6 +1100,17 @@
     <output name="out" type="vector4" default="0.0, 0.0, 0.0, 0.0" />
   </nodedef>
 
+  <!--
+    Node: <bump>
+    Offset the surface normal by a scalar value.
+  -->
+  <nodedef name="ND_bump_vector3" node="bump" nodegroup="houdini">
+    <input name="height" type="float" uiname="Height" uisoftmin="0.0" uisoftmax="1.0" value="0" />
+    <input name="scale" type="float" uiname="Scale" uisoftmin="0.0" uisoftmax="1.0" value="1" />
+    <input name="normal" type="vector3" uiname="Normal" value="0, 0, 0" />
```

Comment:
Ideally the default value should be the `normal` geometric property, rather than a constant value.

Reply:
Thanks, yes will fix this oversight (along with the "houdini" group you pointed out 🤦 )